src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -133,33 +133,21 @@ export class GcsApiService extends GcsService {
   }
 
   getDownloadingFiles() {
-    const items = new FilesDatabase(this.store).data.
-      filter(item => item.status === ItemStatus.DOWNLOADING ||
-        (item.status === ItemStatus.PENDING && item.type === Type.DOWNLOAD));
+    const items = new FilesDatabase(this.store).data.filter(item => item.status === ItemStatus.DOWNLOADING ||
+      (item.status === ItemStatus.PENDING && item.type === Type.DOWNLOAD));
```

Comment:
It looks like these three methods could be refactored into a single method that accepts a `Type` parameter. There already is `getFiles()` which is pretty much that method, minus the refactoring.

Also, there's no need for an intermediate variable `items` here.

Reply:
You are right! this is now refactored